docs/webhooks.md

```diff
@@ -0,0 +1,78 @@
+Webhooks
+========
+
+A webhook object enables you to attach events triggers to Box files and folders. These
+event triggers monitor events on Box objects and notify your application, via HTTP
+requests to a URL of your choosing, when they occur.
+
+* [Create Webhook on File](#create-a-file-webhook)
+* [Create Webhook on Folder](#create-a-folder-webhook)
+* [Get a Webhooks Information](#get-a-webhooks-information)
+* [Get all Webhooks Information](#get-all-webhooks-information)
+* [Update a Webhook](#update-a-webhook)
+* [Delete a Webhook](#delete-a-webhook)
+
+Create a Webhook on a Box File
+------------------------------
+
+Calling
+[`webhooks.create(fileID, objectType, notificationURL, triggerTypes, callback)`](http://opensource.box.com/box-node-sdk/Webhooks.html#create)
+on a file attaches an event trigger with a URL to send notifications to
+
+```js
+client.webhooks.create('759371', FILE, 'https://www.YOURWEBSITE.com', [webhookTriggerType.FILE.UPLOADED, webhookTriggerType.FILE.DOWNLOADED], callback)
+```
+
+alternatively, you can attach a webhook to a folder by calling
+[`webhooks.create(folderID, objectType, notificationURL, triggerTypes, callback)`](http://opensource.box.com/box-node-sdk/Webhooks.html#create)
+
+```js
+client.webhooks.create('15937321', FOLDER, 'https://www.YOURWEBSITE.com', [webhookTriggerType.FOLDER.CREATED, webhookTriggerType.FOLDER.DOWNLOADED], callback)
+```
+
+The notification URL must be a valid HTTPS URL that you specify when you create a
+webhook.
+
+The triggerTypes param is an array of strings. Available options are documented here
+(https://docs.box.com/reference#event-triggers)
+
+
+Get Webhook
+-----------
+
+Retrieve a file or folder's webhook by calling
+[`webhooks.get(webhookID, qs, callback)`](http://opensource.box.com/box-node-sdk/Webhooks.html#get),
+to retrieve a webhook by ID, or
+[`webhooks.getAll(qs, callback)`](http://opensource.box.com/box-node-sdk/Webhooks.html#getAll)
```

Comment:
I would actually split this into two different sections: one for getting a specific webhook and another for getting all webhooks in an enterprise